test/e2e/driver.go

```diff
@@ -75,7 +75,9 @@ func RunE2ETests(authConfig, certDir, host, repoRoot, provider string, orderseed
 		r = append(r, reporters.NewJUnitReporter(path.Join(reportDir, "junit.xml")))
 	}
 	// Run the existing tests with output to console + JUnit for Jenkins
-	ginkgo.RunSpecsWithDefaultAndCustomReporters(&passed, "Kubernetes e2e Suite", r)
+	for i := 0; i < times && passed; i++ {
+		ginkgo.RunSpecsWithDefaultAndCustomReporters(&passed, fmt.Sprintf("Kubernetes e2e Suite run %d of %d", i+1, times), r)
+	}
```

Comment:
Hm. I don't think this would interact well with --report_dir, would it? (It would just overwrite the junit.xml).